hexrd/ui/materials_panel.py

```diff
@@ -276,6 +276,7 @@ def show_overlay_manager(self):
 
     def update_table(self):
         HexrdConfig().update_reflections_tables.emit(self.current_material())
+        HexrdConfig().azimuthal_overlay_modified.emit()
```

Comment:
@psavery My thinking here is that I believe any changes that may update the density value will end up hitting this function so we can emit a modified signal just in case? But my logic may be flawed/misguided so I'm open to changes if needed!

Reply:
Hey @bnmajor! I think it's a little better design for the materials panel to not be aware of other parts of the program, and basically just emit a signal that says "This was modified!", and have other parts of the program listen for that signal to respond.

On that note, there is a `HexrdConfig().active_material_modified` signal. The active material is whatever one is currently displayed in the materials panel, which is usually the only one that gets modified since the user can only see that one in the panel. If you hook into that signal, you will get the same behavior as what you have here, and I think that might be better.

However, there are some cases where things like lattice parameters get modified by calibration. I'm thinking we should add a `HexrdConfig().material_modified(str)` signal, where the `str` is the name of the material that was modified (and we should emit this anytime we change something about the material behind the scenes). But we don't necessarily need to do that for this PR.

Reply:
Awesome, thank you @bnmajor!